README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Go Reference](https://pkg.go.dev/badge/mvdan.cc/gofumpt/format.svg)](https://pkg.go.dev/mvdan.cc/gofumpt/format)
 
-	go install mvdan.cc/gofumpt@latest
+    go install mvdan.cc/gofumpt@latest
```

Comment:
please undo the indentation changes - tabs are fine

Reply:
Sorry for that—I think it was a change automatically introduced by Prettier and that I didn't manually discard. I've now updated things.